.envrc

```diff
@@ -74,6 +74,15 @@ fi
 export LOCAL_CSV_DIR="static/reports"
 export DOCKER_CSV_DIR="/app/$LOCAL_CSV_DIR"
 
+# ----------------------
+# PIPENV LOCALE SETTINGS
+# ----------------------
+# Set the language and locale because macOS has a bug that prevents pipenv from
+# properly detecting the Shell's encoding correctly.
+# source: https://pipenv.pypa.io/en/latest/diagnose/#valueerror-unknown-locale-utf-8
+export LC_ALL='en_US.UTF-8'
+export LANG='en_US.UTF-8'
```

Comment:
Thank you for finding this. It seems like we shouldn't override these environment variables if they are already set. Do you think we could check that before setting defaults?